.github/workflows/build.yml

```diff
@@ -18,6 +18,8 @@ jobs:
         python-version: ["3.9", "3.10", "3.11"]
     steps:
       - uses: actions/checkout@v3
+      - name: Set PYTHONHASHSEED
+        run: echo "PYTHONHASHSEED=42" >> $GITHUB_ENV
```

Comment:
I don't understand why this needs to be done for builds... is it just so te warning below doesnt get raised?

Reply:
Right. We're enforcing that PYTHONHASHSEED == 42 when `SimulationContext` is instantiated. Plenty of pytests in this repo do indeed instantiate so we need to set it here as well (remember that the github jobs are NOT being run in your local env/shell)